orc8r/cloud/configs/analytics.yml

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
We should keep the analytics core service separate from the orc8r analytics implementation. This means this config will reside at `orchestrator.yml`. This is same pattern we use for other core functionality like mconfig. Configurator calls the orchestrator service for orchestrator mconfigs.